lib/presentation/widget/myspace/my_space_widget.dart

```diff
@@ -81,6 +84,41 @@ class _MySpaceWidgetState extends State<MySpaceWidget> {
         builder: (BuildContext context, MySpaceViewModel viewModel) => Scaffold(
               body: Column(
                 children: [
+                  StoreConnector<AppState, MySpaceState>(
+                    converter: (Store<AppState> store) => store.state.mySpaceState,
+                    builder: (context, state) {
+                      return state.selectMode == SelectMode.ACTIVE ? ListTile(
+                        leading: SvgPicture.asset(
+                          imagePath.icSelectAll,
+                          width: 28,
+                          height: 28,
+                          fit: BoxFit.fill,
+                          color: state.documentList.every((value) => value.selectMode == SelectMode.ACTIVE) ?
```

Comment:
since you're using this in many places, please create an extension for documentList to detect the list has active mode item or not.

Reply:
Sure great idea